include/oneapi/dpl/pstl/glue_algorithm_ranges_impl.h

```diff
@@ -526,6 +577,105 @@ struct __max_element_fn
 
 inline constexpr __internal::__max_element_fn max_element;
 
```

Comment:
```suggestion
// [alg.minmax_element]
```
I'll stop adding these suggestions for other instances, but it seems like this was the previous convention, and we should be consistent.

Reply:
Good catch, thank you.
Added "titles" for the all similar places.